.eslintrc.json

```diff
@@ -1,4 +1,5 @@
 {
+	"extends": "eslint:recommended",
```

Comment:
I would really prefer we explicitly list rules than import a black box set like this.